### PR TITLE
CASMTRIAGE-4237:  Clarify the BGP reset procedures to make sure both VRFs are handled

### PR DESCRIPTION
**Pull request by johren-hpe**
# Description

The clear commands in Check_BGP_Status_and_Reset_Sessions.md only affect the default VRF.   Needed specify the clear commands that will work for each VRF.

Tested these commands on drax for mellanox and mug for Aruba.

# Checklist Before Merging

- [  ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [  ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by keopp-hpe — APPROVED** [review]

**Review by ekamin-hpe — COMMENTED** [review]
Not sure why @Cray-HPE/system-admin-toolkit-admin was requested to review as a code owner.

File: operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md [hunk 1]
```diff
@@ -85,6 +85,12 @@ The following procedures may not resolve the problem after just one attempt. In
         ssh admin@sw-spine-001.hmn
         ```
 
+    1. (`sw-spine>`) Enter enable mode
```
Comment (nrockershousen) [inline operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md:88]:
```suggestion
    1. (`sw-spine#`) Enter enable mode.
```

File: operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md [hunk 2]
```diff
@@ -99,9 +105,18 @@ The following procedures may not resolve the problem after just one attempt. In
 
     1. (`sw-spine#`) Clear the BGP sessions.
 
+        There are two VRF's that may need to be cleared. Clear the VRF that has the `Idle` session state.
+
+        Default VRF
```
Comment (nrockershousen) [inline operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md:110]:
```suggestion
        Default VRF:
```

File: operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md [hunk 3]
```diff
@@ -99,9 +105,18 @@ The following procedures may not resolve the problem after just one attempt. In
...
+
         ```text
-        enable
-        clear ip bgp all
+        clear ip bgp vrf default all
+        ```
+
+        Customer VRF
```
Comment (nrockershousen) [inline operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md:116]:
```suggestion
        Customer VRF:
```

File: operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md [hunk 4]
```diff
@@ -228,8 +243,18 @@ The following procedures may not resolve the problem after just one attempt. In
 
     1. (`sw-spine#`) Clear the BGP sessions.
 
+        There are two VRF's that may need to be cleared. Clear the VRF that has the `Idle` session state.
+
+        Default VRF
```
Comment (nrockershousen) [inline operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md:248]:
```suggestion
        Default VRF:
```

File: operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md [hunk 5]
```diff
@@ -228,8 +243,18 @@ The following procedures may not resolve the problem after just one attempt. In
...
+
+        ```text
+        clear bgp vrf default *
+        ```
+
+        Customer VRF
```
Comment (nrockershousen) [inline operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md:254]:
```suggestion
        Customer VRF:
```

**Review by nrockershousen — APPROVED** [review]

**Comment by rustydb** [comment]
> Not sure why @Cray-HPE/system-admin-toolkit-admin was requested to review as a code owner.

Will look into.

Seems somehow github is getting confused. The `/operations/sat` dir wasn't touched at all here.

**Review by rustydb — APPROVED** [review]

**Comment by rustydb** [comment]
> > Not sure why @Cray-HPE/system-admin-toolkit-admin was requested to review as a code owner.
> 
> Will look into.
> 
> Seems somehow github is getting confused. The `/operations/sat` dir wasn't touched at all here.

@ekamin-hpe I filed this.. hopefully resolved soon https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5382

**Pull request merged**